app/config/routing.yml

```diff
@@ -9,9 +9,9 @@ app:
     type:     annotation
     prefix:   /{_locale}
     requirements:
-        _locale: %app_locales%
+        _locale: '%app_locales%'
```

Comment:
Hm, seems Symfony plugin's autocomplete work incorrectly when `%` in quotes (it's doubled `%`), so I reverted changes for parameters, thanks @xabbuh !